[docs] removed duplicate titles and icon added for voyager (#16421)
yugabyte/yugabyte-db: docs/content/preview/migrate/install-yb-voyager.md

---
title: Install yb-voyager
headerTitle: Install
linkTitle: Install

> description: Explore the prerequisites, YugabyteDB Voyager installation, and so on.

Reviewer: ```suggestion
description: Prerequisites and installation instructions for YugabyteDB Voyager.
```
